ff2mpv.py

```diff
@@ -1,17 +1,57 @@
 #!/usr/bin/env python3
-import sys, struct, json
+import sys, struct, json, tempfile
```

Comment:
JFYI, I'm going to work on adding continuous integration to this repo (including linting of the ruby and python clients). Once that goes through, we'll rebase this PR to make sure the changes here are idiomatic. No changes needed on your part here until that's done :slightly_smiling_face: